database/gorm/test_models.go

```diff
@@ -34,9 +34,21 @@ func (u *User) DispatchesEvents() map[ormcontract.EventType]func(ormcontract.Eve
 				if name.(string) == "event_creating_name" {
```

Comment:
Instead of this, would it be better to cast it? Because if this is not asserted, it will panic.

Reply:
Sure, I'll optimize it in another PR.